src/App.tsx

```diff
@@ -1,28 +1,53 @@
 import React from 'react';
 import './App.css';
 import { useForm, SubmitHandler } from 'react-hook-form';
+import { ApolloClient, InMemoryCache, gql } from '@apollo/client';
 
 type FormValues = {
   email: string;
   password: string;
 };
 
 function App() {
+  const [loginError, setLoginError] = React.useState<string>();
+
   const {
     register,
     handleSubmit,
     formState: { errors },
   } = useForm();
 
-  const onSubmit: SubmitHandler<FormValues> = (data) => {
-    console.log(data);
+  const onSubmit: SubmitHandler<FormValues> = async (data) => {
+    setLoginError('');
```

Comment:
aqui vc pode passar undefined

```suggestion
    setLoginError(undefined);
```